docs/user-guide/connectors/index.md

```diff
@@ -52,14 +52,20 @@ protocol][cdp]. `chrome` is one of these `connector`s.
 
 The set of settings specific for them are:
 
-* `useTabUrl`: Indicates if the browser should navigate first to a
+* `defaultProfile (boolean)`: Indicates if the browser should use the
```

Comment:
Should this be `forceDefaultProfile`?

Reply:
Yes, it got updated later but because we are still debating if it should be a new parameter or `userDataString: false` I haven't updated.
Current discussion in https://github.com/GoogleChrome/chrome-launcher/pull/48#issuecomment-343055040